lib/rest2/methods.js

```diff
@@ -87,7 +87,15 @@ const RESTV2_MOCK_METHODS = {
   '/v2/auth/r/ext/pay/invoices': 'invoice_list',
   '/v2/auth/w/ext/pay/invoice/complete': 'invoice_complete',
   '/v2/auth/r/ext/pay/deposits/unlinked': 'deposits_unlinked',
-  '/v2/auth/r/movements/info': 'movement_info.{id}'
+  '/v2/auth/r/movements/info': 'movement_info.{id}',
```

Comment:
can we move this up, to have all 'ext/pay' endpoints together?

Reply:
Done!